README.md

```diff
@@ -80,6 +80,17 @@ data_importer.wait_until_complete
  => Success()
 data_importer.instance_hrid
  => Success("in00000000010")
+
+# Create a Holdings record
+holdings_client = client.holdings(instance_id: "99a6d818-d523-42f3-9844-81cf3187dbad")
+holdings_client.create(permanent_location_id: "1b14e21c-8d47-45c7-bc49-456a0086422b", holdings_type_id: "996f93e2-5b5e-4cf2-9168-33ced1f95eed")
+ => {
+    "id" => "581f6289-001f-49d1-bab7-035f4d878cbd",
+    "_version" => 1,
+    "hrid" => "ho00000000065",
+    "holdingsTypeId" => "996f93e2-5b5e-4cf2-9168-33ced1f95eed"
+    ...
+ }
```

Comment:
Do you think we'll be instantiating a `Holdings` instance and then interacting with it more than once? If so, this design makes sense to me. If not, we may want to consider a change like https://github.com/sul-dlss/folio_client/commit/ad68843e9fb18ac98ce509f006ddfb1c488e208a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R75

Reply:
Yes, I suspect (but not certain) that we're going to need additional methods for reading and/or checking the existence of holdings records.

That being said, I also think that once all of the initial Folio work is done, we're probably going to want to clean up this gem for these sorts of issues.